pkg/resources/custom-domain/prometheusRules.go

```diff
@@ -44,9 +44,39 @@ func customDomainCRErrorState(installationName string, namespace string, err str
 					"sop_url": resources.SopUrlRHOAMServiceDefinition,
 					"message": fmt.Sprintf(fmt.Sprintf("Error configuring custom domain, please refer to the documetaion to resolve the error. Found error: %s", err)),
 				},
-				Expr:   intstr.FromString(fmt.Sprintf("%s_custom_domain{customDomain='active'} > 0", installationName)),
+				Expr:   intstr.FromString(fmt.Sprintf("%s_custom_domain{active='false'} > 0", installationName)),
```

Comment:
I am not sure if this is doing what it is meant to be doing. This needs to be verified that is still working as expected.